azerothcore/azerothcore-wotlk
# feat(Core/Mail): Server mail

## Changes Proposed:
-  Introduce mail templates to be set and sent at player login. Kind of how blizzard handles anniversary gifts.

## Tests Performed:

- works ingame


## How to Test the Changes:


1. Populate mail_server_template table with something.
2. Login, see that if the character meets the requirements, you will get the mail
3. Relog and see that you will not get the mail again since it was registered in mail_server_character


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
